test/model/test/common/include/model/test/wiredtiger_util.h

```diff
@@ -209,36 +213,43 @@ wt_rollback_to_stable(WT_CONNECTION *conn)
 }
 
 /*
- * wt_model_assert_equal --
+ * wt_model_assert_equal_with_labels --
  *     Check that the two values are the same.
  */
-#define wt_model_assert_equal(a, b)                                               \
+#define wt_model_assert_equal_with_labels(a, b, label_a, label_b)                 \
     {                                                                             \
-        auto ret_a = a;                                                           \
-        auto ret_b = b;                                                           \
+        auto ret_a = (a);                                                         \
+        auto ret_b = (b);                                                         \
         if (ret_a != ret_b) {                                                     \
             std::cerr << std::endl;                                               \
             std::cerr << "Assertion failure in " << __PRETTY_FUNCTION__           \
                       << ": Values are not equal. " << std::endl;                 \
             std::cerr << "  Location: " << __FILE__ ":" << __LINE__ << std::endl; \
-            std::cerr << "  " << #a << ": " << ret_a << std::endl;                \
-            std::cerr << "  " << #b << ": " << ret_b << std::endl;                \
+            std::cerr << "  " << (label_a) << ": " << ret_a << std::endl;         \
+            std::cerr << "  " << (label_b) << ": " << ret_b << std::endl;         \
```

Comment:
What did Coverity dislike about this? I think the code was easier to read and simpler before.

Reply:
I agree that it was easier to read before. Coverity didn't like that the code didn't use `std::move` and created two issues for each usage of `wt_model_assert`, which was a lot of issues! I didn't want `std::move` to show up in the output to `cerr`, so I split the macro accordingly.

Reply:
Ouch. Fair enough. Thanks for the explanation.